src/System.IO.IsolatedStorage/tests/System.IO.IsolatedStorage.Tests.builds

```diff
@@ -3,15 +3,9 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.IO.IsolatedStorage.Tests.csproj">
-      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Even though this really doesn't do anything anymore since the build doesn't depend on OSGroup, @weshaggard made the comment that we should probably keep the OSGroup when we are testing Desktop just to be consistent with the rest of the repo